### DISCUSSION TITLE
I cannot send data larger than 1500 bytes。

### DISCUSSION THREAD

**Discussion by Jun1172** [eProsima/Fast-DDS#4499] [Q&A]
hello
I can send shorter data, but after the data exceeds 1500, I cannot send it anymore
I found that when UDP has packets, it cannot be received。

I have set xxCdrAux.hpp max_cdr_typesize=5000,and FASTCDR_VERSION_MAJOR == 2
Can you help me ,thanks!

**Comment by Jun1172** [comment]
More Information：
1.I use it in WSL, the system is UBUNTU
2.If used in Ubuntu, there is no problem, you can send 9000 bytes
3.If used under Windows, it cannot exceed 1514

**Comment by Jun1172** [comment]
I have resolved it, it's WSL's MTU restriction

**Comment by elianalf** [comment]
Hello, thanks for writing. If the issue is resolved could you please close the discussion?
